### PR TITLE
add support for lhttpc pools

### PR DESCRIPTION
**Pull request by juantascon**

### CODE REVIEW

File: include/erlcloud_aws.hrl [hunk 1]
```diff
@@ -51,6 +51,7 @@
           cloudtrail_raw_result=false::boolean(),
           http_client=lhttpc::erlcloud_httpc:request_fun(), %% If using hackney, ensure that it is started.
           hackney_pool=default::atom(), %% The name of the http request pool hackney should use.
+          lhttpc_pool=default::atom(), %% The name of the http request pool lhttpc should use.
```
Comment (nalundgaard) [inline include/erlcloud_aws.hrl]: Is `default` correct, or should this be `undefined`?

Comment (motobob) [inline include/erlcloud_aws.hrl]: indeed, it should be `undefined` given the way `request_lhttpc()` is written

**Comment by juantascon** [comment]
fixed!

**Comment by nalundgaard** [comment]
👍 Thanks for your contribution.

**Pull request merged**